File: prima.js
function hitungBilanganPrima(awal, akhir) {
    let jumlahPrima = 0;
  
    for (let i = awal; i <= akhir; i++) {
      if (isPrime(i)) {
        jumlahPrima += i;
      }
    }
  
    return jumlahPrima;
  }
  
  function isPrime(num) {
    if (num <= 1) return false;
    for (let i = 2; i <= Math.sqrt(num); i++) {
      if (num % i === 0) return false;
    }
    return true;
  }
  
  const inputAwal = 1;
  const inputAkhir = 12;
  const hasil = hitungBilanganPrima(inputAwal, inputAkhir);
  console.log(`Jumlah bilangan prima dari ${inputAwal} sampai ${inputAkhir} adalah: ${hasil}`);

File: prima.py
def is_prime(n):
    if n <= 1:
        return False
    for i in range(2, int(n**0.5) + 1):
        if n % i == 0:
            return False
    return True

def next_prime(start):
    num = start + 1  
    while True:
        if is_prime(num):
            return num
        num += 1

input_number = int(input("Masukkan bilangan: "))
output_prime = next_prime(input_number)

print(f"Bilangan prima setelah {input_number} adalah {output_prime}.")